rcsb/app/file/Sessions.py

```diff
@@ -0,0 +1,478 @@
+# file: Sessions.py
+# author: James Smith 2023
+
+import asyncio
+import json
+import os
+import sys
+import time
+import logging
+import uuid
+import typing
+from rcsb.app.file.ConfigProvider import ConfigProvider
+from rcsb.app.file.PathProvider import PathProvider
+from rcsb.app.file.KvRedis import KvRedis
+from rcsb.app.file.KvSqlite import KvSqlite
+
+logging.basicConfig(level=logging.INFO)
+
+# session maintenance for one upload
+# includes file locking and optional resumability API
+# functions - upload helper functions, database functions, locking functions, session placeholder functions, bulk maintenance function
+
+
+class Sessions(object):
+    # statelessly invoked once per chunk of every upload
```

Comment:
Sorry, do you mean for every chunk, a new `Sessions` class is instantiated?

Reply:
Yes, it must rejoin the previous session. It's a REST application.

Reply:
Ah, so this is on the server side, I see.